src/applications/letters/utils/helpers.jsx

```diff
@@ -356,6 +414,11 @@ export const getFormattedDate = dateTime => {
   return formatDateShort(stripOffTime(dateTime));
 };
 
+export function getFriendlyBenefitSummaryLabels(option, isVeteran) {
```

Comment:
For now, `isVeteran` is hard-coded in the new `VeteranBenefitSummaryOptions` component. I feel this is an okay decision that gives us flexibility to pull it from app state later if we want to start offering options to non-Veteran dependents.